packages/graphql/tests/performance/performance.ts

```diff
@@ -137,11 +71,36 @@ async function main() {
         await schemaPerformance();
     } else if (process.argv.includes("--subgraph-schema")) {
         await subgraphSchemaPerformance();
+    } else if (process.argv.includes("--translation")) {
+        await translationPerformance();
     } else {
         await queryPerformance();
     }
 }
 
+async function translationPerformance() {
+    let runs = 100;
+    const runsArg = getArgumentValue("--runs");
+    if (runsArg) {
+        const parsedRuns = Math.trunc(Number(runsArg));
+        if (!parsedRuns) throw new Error("--runs require a positive number");
```

Comment:
I think `Math.trunc(Number("-1000")` would still be truthy in this check.

Reply:
good point